testing/integration/install_test.go

```diff
@@ -338,32 +338,41 @@ func TestInstallUninstallAudit(t *testing.T) {
 		return waitForAgentAndFleetHealthy(ctx, t, fixture)
 	}, time.Minute, time.Second, "agent never became healthy or connected to Fleet")
 
-	agentID, err := getAgentID(ctx, fixture)
-	require.NoError(t, err, "error getting the agent inspect output")
-	require.NotEmpty(t, agentID, "agent ID empty")
+	t.Run("run uninstall", testUninstallAuditUnenroll(ctx, fixture, info))
```

Comment:
Am I blind or is this the exact same test logic before that was passing before this bug fix? Was this test actually catching the problem? Or was something changed I just can't see yet?

Reply:
OK nevermind reading the wrong file, it's called in a new place now.